.ci/lib.sh

```diff
@@ -327,6 +327,54 @@ build_install_parallel() {
 	rm -rf "$gnu_parallel_dir" "$gnu_parallel_tar_pkg"
 }
 
+build_install_cmake() {
+	local version=$(get_version "externals.cmake.version")
+	local url=$(get_version "externals.cmake.url")
+	[ "${url}" != "null" ] && [ "${version}" != "null" ]
+	local file="cmake-${version}.tar.gz"
+	local dir="cmake-${version}"
+	local download="${url}/releases/download/v${version}/${file}"
+
+	pushd /root
```

Comment:
This assumes the script is running as `root` which won't be the case for the CI and shouldn't be the case for devs either. How about something like:

```sh
build_dir=$(mktemp -d)
pushd "$build_dir" &>/dev/null

    :

popd &>/dev/null
rm -rf "$build_dir"
```